.github/CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ This release contains contributions from (in alphabetical order):
 * Update Kokkos support to 4.0.01.
   [(#439)] (https://github.com/PennyLaneAI/pennylane-lightning/pull/439)
 
+* Lightning supports SparseHamiltonian natively (without Kokkos & Kokkos-kernels).
```

Comment:
I consider that our support, in terms of KokkosKernels, for SparseHamiltonian is already native.
I propose something like this:
```suggestion
* Add native support to sparse Hamiltonians in the absence of Kokkos & Kokkos-kernels.
```